packages/loot-core/src/server/accounts/qif2json.ts

```diff
@@ -1,19 +1,44 @@
-export default function parse(qif, options) {
-  var lines = qif.split('\n'),
-    line = lines.shift(),
-    type = /!Type:([^$]*)$/.exec(line.trim()),
-    data = {},
-    transactions = (data.transactions = []),
-    transaction = {};
+type Division = {
+  category?: string;
+  subcategory?: string;
+  description?: string;
+  amount?: number;
+};
 
-  options = options || {};
+type Transaction = {
```

Comment:
Maybe prefix these names to clarify that they aren’t Actual transactions/divisions?
```suggestion
type QIFTransaction = {
```